dotcom-rendering/src/components/Picture.tsx

```diff
@@ -321,15 +321,11 @@ export const Picture = ({
 	orientation = 'landscape',
 	onLoad,
 }: Props) => {
-	const ref = useRef<HTMLImageElement>(null);
 	const [loaded, setLoaded] = useState(false);
-
-	useEffect(() => {
-		if (!ref.current) return;
-
-		if (ref.current.complete) return setLoaded(true);
-		ref.current.addEventListener('load', () => setLoaded(true));
-	}, [ref]);
+	const ref = useCallback((node: HTMLImageElement) => {
+		if (node.complete) return setLoaded(true);
```

Comment:
Aside from it being a one liner returning from a setter doesn't have any meaning as setters return void. Perhaps this has clearer intent?:

```suggestion
		if (node.complete) {
			setLoaded(true);
			return;
		}
```